lib/jwt/token/asymmetric_key_configuration.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module JWT
+  class Token
+    class AsymmetricKeyConfiguration
+      class PublicKeySet
+        def initialize(keys)
+          @keys = keys
+        end
+
+        def verify(digest, signature, data)
+          @keys.any? { |key| key.verify(digest, signature, data) }
+        end
+      end
+
+      attr_accessor :authorized_keys, :private_key
+
+      def initialize(key_class)
+        @key_class = key_class
+      end
+
+      def authorized_keys_file=(file_path)
+        self.authorized_keys =
+          File.read(file_path)
```

Comment:
Do we want keep keys in file?

Reply:
It seems like a viable options (these are only public keys anyway).

Think `.ssh/authorized_keys`